### PR TITLE
CMS:update cms-tools-vm-image-Run2011A.xml

### PR DESCRIPTION
**Pull request by AnxhelaDani**

### CODE REVIEW

File: invenio_opendata/testsuite/data/cms/cms-tools-vm-image-Run2011A.xml [hunk 1]
```diff
@@ -74,7 +74,7 @@
     </datafield>
     <datafield tag="538" ind1=" " ind2=" ">
       <subfield code="a">Use this with a Micro-CernVM available at</subfield>
-      <subfield code="u">http://cernvm.cern.ch/portal/downloads</subfield>
+      <subfield code="w">http://cernvm.cern.ch/portal/downloads</subfield>
```
Comment (tiborsimko) [inline invenio_opendata/testsuite/data/cms/cms-tools-vm-image-Run2011A.xml]: This change of subfields seems wrong; 538 $w should contain record ID values, see the field rule book:

- https://github.com/cernopendata/opendata.cern.ch/blob/master/invenio_opendata/base/recordext/fields/opendata.cfg#L154-L160

Comment (AnxhelaDani) [inline invenio_opendata/testsuite/data/cms/cms-tools-vm-image-Run2011A.xml]: The link at `538__ $u` will appear on the record page after the updates on `opendata.cfg` file (PR 1024)

Comment (tiborsimko) [inline invenio_opendata/testsuite/data/cms/cms-tools-vm-image-Run2011A.xml]: Yes, but you should keep URLs in $u and not change it to $w as your PR does... that's what I meant in the above comment.

Comment (AnxhelaDani) [inline invenio_opendata/testsuite/data/cms/cms-tools-vm-image-Run2011A.xml]: But I change it again at $u.. sorry I'll check again

**Comment by tiborsimko** [comment]
Thanks, squashed and merged as b7084587c49dc58b2604212a497e85fa242ba8dd.

**Pull request closed without merging**